docker-for-azure/faqs.md

```diff
@@ -5,7 +5,23 @@ title: Docker for Azure Frequently asked questions (FAQ)
 toc_max: 2
 ---
 
-## FAQ
+## Stable and beta channels
```

Comment:
Is it worth linking to the other page that has the links for the stable and beta channel, from the FAQ?